vsphere-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -9,15 +9,15 @@
 #vsphere_port: 443
 #vsphere_datacenter_name: Datacenter
 #vsphere_resource_pool_name: Resources
-#vsphere_auto_placement: true
+#vsphere_auto_placement: false
 
 # Template
-#template: "CentOS7_1406_tmpl"
+#template: "CentOS-7-x86_64-1406"
 
 # Names of components
 #management_network_name: "Management"
-#external_network_name: "Application"
-#server_name: "cloudify-manager-server"
+#external_network_name: "DMZ"
+#server_name: "centosmanager"
```

Comment:
why change the name of the server? The previous one actually makes more sense to me.